### PR TITLE
Set Default to Dark Mode #1148

### PR DESCRIPTION
**Pull request by gourangi-taware**
I have added light-mode as "dark" in each of the pages so that the default theme becomes dark. For making the light theme consistent while navigating through pages, I have made some changes in app.js as follows
image

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Once you switch from dark mode to light mode the navbar becomes white completely.
image
here's a sample

**Comment by gourangi-taware** [comment]
Okay, I'll update the code accordingly.

**Comment by gourangi-taware** [comment]
I checked the code and the integration, but the case where the navbar is becoming white is only for the Experience page. It is working fine with all the other pages. I made changes only in app.js (globally) and only changed the theme for respective pages. I am trying to understand the reason but I am unable to find the exact error. Can you please guide me to resolve the issue?  Following are some of the samples of the working:

image
image
image

**Review by anushbhatia — COMMENTED** [review]
Check the css code for experience.css 
https://github.com/smaranjitghose/awesome-portfolio-websites/pull/992/files this might have caused this issues

**Comment by gourangi-taware** [comment]
I have made the necessary changes and committed the changes
image
image
Please review the changes

File: assets/css/experience.css [hunk 1]
```diff
@@ -666,7 +666,7 @@ ul ol {
 }
 a:link {
   text-decoration: none !important;
-  color: white;
+  /* color: white; */
```
Comment (anushbhatia) [inline assets/css/experience.css:669]: remove unused text it won't be any good here.

**Review by anushbhatia — CHANGES_REQUESTED** [review]

**Comment by gourangi-taware** [comment]
I have done the required changes. Please review

**Comment by anushbhatia** [comment]
great work @gourangi-taware  and keep it up.

**Pull request merged**